src/docquery/ext/config.py

```diff
@@ -0,0 +1,34 @@
+# coding=utf-8
+# Copyright 2010, The Microsoft Research Asia LayoutLM Team authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" LayoutLM model configuration"""
+from transformers import LayoutLMConfig
+
+
+class LayoutLMTokenClassifierConfig(LayoutLMConfig):
+    model_type = "layoutlm-tc"
+
+    def __init__(
+        self,
+        # New stuff added for DocQuery
```

Comment:
nit: Can you include a docstring from https://github.com/huggingface/transformers/blob/0a632f076d6b275690176b79c64c5559e1240b05/src/transformers/models/layoutlm/configuration_layoutlm.py#L40 and include explanations for `token_classification`, `token_classifier_reduction`, and `token_classifier_constant` there? @jeffsrobertson Are you able help with the explanations?

Reply:
Good idea. Added the basic parts and left TODOs where I need @jeffsrobertson's help.